<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionAdd_User"/>
    <addaction name="actionDelete_User"/>
    <addaction name="actionChange_Password"/>
    <addaction name="actionLogin"/>
   </widget>
   <widget class="QMenu" name="menuStudent">
    <property name="title">
     <string>Student</string>
    </property>
    <addaction name="actionOpen_Form1"/>
   </widget>
   <widget class="QMenu" name="menuFees">
    <property name="title">
     <string>Fees</string>
    </property>
    <addaction name="actionOpen_Form2"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionOpen_Form3"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuStudent"/>
   <addaction name="menuFees"/>
   <addaction name="menuReport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_User">
   <property name="text">
    <string>Add_User</string>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete_User</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change_Password</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionOpen_Form1">
   <property name="text">
    <string>Open_Form1</string>
   </property>
  </action>
  <action name="actionOpen_Form2">
   <property name="text">
    <string>Open_Form2</string>
   </property>
  </action>
  <action name="actionOpen_Form3">
   <property name="text">
    <string>Open_Form3</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
